models/models/sample.py

```diff
@@ -15,14 +15,16 @@ class SampleInternal(SMBase):
     """Internal model for a Sample"""
 
     id: int
-    external_id: str
+    external_ids: dict[str, str]
     meta: dict
     project: int
     type: str | None
     participant_id: int | None
     active: bool | None
     author: str | None = None
 
+    audit_log_id: int | None = None
```

Comment:
This is actually more fallout from PR #752, so the getHistoryOfSample endpoint is likely already broken — `SampleTable.get_history_of_sample()` returns an `audit_log_id` field that its `….from_db()` call will choke on. So theoretically this could be a separate PR! (Or the alternative fix would be to avoid the problem by dropping `audit_log_id` from the fields selected / data returned.)

Reply:
Yep. Now removed again.